src/open_deep_research/tdd/vdr/data_room.py

```
@@ -0,0 +1,292 @@
+    
+    def load(self, load_dir: Path) -> None:
+        """Load the VDR from disk.
+        
+        Args:
+            load_dir: Directory to load VDR data from
+        """
+        # Load domain reports
+        reports_dir = load_dir / "reports"
+        if reports_dir.exists():
+            for report_file in reports_dir.glob("*.json"):
+                with open(report_file, "r") as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)